sdk/sdk/OBAApplication.h

```diff
@@ -0,0 +1,44 @@
+//
+//  sdk.h
+//  sdk
+//
+//  Created by Dima Belov on 4/25/15.
+//  Copyright (c) 2015 One Bus Away. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "OBAReferencesV2.h"
```

Comment:
These imports are needed to here for the same reason they were in OBAApllicationDelegate.  Sure, we could do that, but then we'd have to add a bunch of imports everywhere else.  Was out of scope for this change.